docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,303 @@
+    order, where the polygons are filled inward if they have a [clockwise](@ref GeometryOps.isclockwise) winding order and 
+    outward if they have a counterclockwise winding order.
+    
+    Hopefully, these are details that you'll never have to deal with.  But it is good to know.
+
+````@example creating_geometry
+hole = GI.LinearRing(GI.getpoint(multipoint))
+polygon1 = GI.Polygon([ring1, hole]; crs)
+````
+
+Shift `polygon1` to the right, to avoid plotting over our earlier results.
+
+````@example creating_geometry
+xoffset = 50.
+yoffset = 0.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+polygon1 = GO.transform(f, polygon1)
+plot!(polygon1)
+fig
+````
+
+`Polygon`s can also be grouped together as a `MultiPolygon`.
+
+````@example creating_geometry
+r = 5
+x = cos.(reverse(ϴ)) .* r .+ xoffset
+y = sin.(reverse(ϴ)) .* r .+ yoffset
+ring2 =  GI.LinearRing(GI.Point.(zip(x,y)))
+polygon2 = GI.Polygon([ring2])
+multipolygon = GI.MultiPolygon([polygon1, polygon2]; crs)
+````
+
+Shift `multipolygon` up, to avoid plotting over our earlier results.
+
+````@example creating_geometry
+xoffset = 0.
+yoffset = 50.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+multipolygon = GO.transform(f, multipolygon)
+plot!(multipolygon)
+fig
+````
+
+Great, now we can make `Points`, `MultiPoints`, `Lines`, `LineStrings`, `Polygons` (with holes), and `MultiPolygons` and modify them using [`CoordinateTransformations`] and [`GeometryOps`].
+
+But where does the `crs` information come in? To show this, we need to use `GeoMakie` that can interpret the `crs` information that we've included with our geometries.
+
+Now specify the source and destination projections for our map. Remember that the very first thing we did was select our source coordinate system.
+
+````@example creating_geometry
+source = crs;
+dest = "+proj=natearth2" #see [https://proj.org/en/9.4/operations/projections/natearth2.html]
+````
+
+Open the Natural Earth continental outlines that are bundled with GeoMakie, and also available from https://www.naturalearthdata.com/.
+
+````@example creating_geometry
+land_path = GeoMakie.assetpath("ne_110m_land.geojson")
+````
+
+Read the land polygons into a `GeoJSON.FeatureCollection`.
+
+````@example creating_geometry
+land_geo = GeoJSON.read(read(land_path, String))
+````
+
+create a figure with a `GeoAxis` from GeoMakie, that can handle the projections between different CRSs.
+
+````@example creating_geometry
+fig = Figure(size=(1000, 500));
+ga = GeoAxis(
+    fig[1, 1];
+    source=crs, # `source` and `dest` set the CRS
+    dest=dest,
+    xticklabelsvisible = false,
+    yticklabelsvisible = false,
+);
+nothing #hide
+````
+
+Plot `land` for context.
+
+````@example creating_geometry
+poly!(ga, land_geo, color=:black)
+fig
+````
+
+Now let's plot a `Polygon` like before, but now on coordinate reference system (CRS) that is different from our data
```

Comment:
You say here that the CRS is different from our data.. but then on line 212 you say we will "plot geometries on the same figure that are in a different Coordinate Reference System (CRS)". So how are these different?